src/main/java/gil/server/controllers/StaticFileOptionsController.java

```diff
@@ -11,7 +10,7 @@
                 String uri = request.getURI();
 
                 if(StaticFileUtils.staticFileExists(uri)) {
-                    String allow = "Allow: OPTIONS, GET";
+                    String allow = "OPTIONS, GET";
```

Comment:
It'd be better if you do this programmatically. Not a requirement but a nice to have. Upto you if you want to work on it

Reply:
I agree and added a function to programmatically determine the Allow header to the Router class.